editor/editor_properties_array_dict.cpp

```diff
@@ -655,6 +655,10 @@ void EditorPropertyArray::_reorder_button_up() {
 	reorder_mouse_y_delta = 0.0f;
 
 	Input::get_singleton()->set_mouse_mode(Input::MOUSE_MODE_VISIBLE);
+
+	if (!reorder_selected_button) {
+		return;
```

Comment:
Hmm... Not sure about this. The only place `reorder_selected_button` is set is in `_reorder_button_down` and `_reorder_button_up`, so either `_reorder_button_up` is getting called twice in a row, or the button somehow becomes null after `_reorder_button_down` (but on `_reorder_button_down` the mouse gets captured so there's not much the user can do). If it fixes the crashing, that's something I guess, but I might try debugging this later see if I can figure anything out.

Reply:
Also, I can reproduce the crash in 3.3, where this code doesn't exist.

Reply:
Well it DOES fix some crash, so maybe mine is unrelated to the original issue.